nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -481,7 +481,7 @@ in
                 + lib.optionalString (isx86_32 || isx86_64) "-Xbcj x86"
                 # Untested but should also reduce size for these platforms
                 + lib.optionalString (isAarch32 || isAarch64) "-Xbcj arm"
-                + lib.optionalString (isPowerPC) "-Xbcj powerpc"
+                + lib.optionalString (isPower32BigEndian) "-Xbcj powerpc"
```

Comment:
I checked and the `xz` man page says it supports

```
PowerPC         4       Big endian only
```

While it doesn't explicitly say 64bit is not supported, it seems that way. So this seems correct after all!